credible/dapp-overview.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Credible Layer dApp'
+description: 'Overview of the Credible Layer dApp features and capabilities'
+---
+
+## What is the Credible Layer dApp?
+
+The Credible Layer dApp provides a comprehensive interface for managing smart contract projects and their associated assertions. It serves as the primary platform for developers and users to interact with the Credible Layer security infrastructure.
+
+## Core Features
+
+### Project Explorer
+
+The dApp's explorer feature enables anyone to search for and analyze projects and their associated assertions, creating transparency about which protocols are protected and how.
+
+**Key capabilities:**
+- Browse all projects on the Credible Layer
+- Search for specific projects by name or address
+- View detailed information about linked assertions for each project
+- Assess a project's security posture before interacting with it
+
+![Explorer](/images/explorer_screenshot_dapp.png)
+
+This transparency allows users to make informed decisions about where to deploy capital, knowing exactly which security measures protect their funds.
+
+### Project Management
+
+For protocol teams, the dApp provides tools to create and manage projects:
+
+**Creating Projects:**
+- Define project details (name, description, target chain)
+- Link contract addresses to your project
+- Ownership verification ensures only contract owners can manage projects
+
+**Managing Assertions:**
+- Submit assertions for review
+- Link assertions to specific contracts
+- Monitor assertion status and activity
+- Update or deactivate assertions as needed
+
+### Authentication
+
+The dApp uses OAuth-based authentication integrated with the `pcl` CLI, ensuring secure access to project management features. Authentication is handled through wallet connection, maintaining alignment with Web3 security practices.
+
+## Who Uses the dApp?
```

Comment:
This is weird and nobody will read it.

I want to read the docs about the various featuers of the dapp, not about who will use it.

You can use the context in the light of "what can i do with the dapp" and add it as comments 
for various tabs